common/fe-ste-models.go

```diff
@@ -50,7 +50,8 @@ const (
 
 	//  this is the perm that AzCopy has used throughout its preview.  So, while we considered relaxing it to 0666
 	//  we decided that the best option was to leave it as is, and only relax it if user feedback so requires.
-	DEFAULT_FILE_PERM = 0644
+	DEFAULT_FILE_PERM     = 0644  // the os package will handle base-10 for us.
+	HEX_DEFAULT_FILE_PERM = 0x644 // when making unix syscalls directly, hex form is expected.
```

Comment:
we should not need to define two and 0x644 is not same as octal 0644.
What we need is just the octal rep i.e. DEFAULT_FILE_PERM and use it in all places.
What the syscall (or any other API) would take is an integer and these are just convenience literal representation in octcal and hex (and decimal).